src/css/skins/seven.less

```diff
@@ -1,308 +1,85 @@
-        }
-        .jw-knob {
-            margin-bottom: -0.3em;
-        }
-    }
-
-    /* Volume slider container */
-    .jw-volume-tip {
-        width: 100%;
-        left: -45%;
-        padding-bottom: .7em;
-    }
-
-    /* Color of video's duration time */
-    .jw-text-duration {
-        color: #666F82;
-    }
-    
-    /* Icon borders on the right side of the controlbar */
-    .jw-controlbar-right-group {
-        .jw-icon-tooltip:before,
-        .jw-icon-inline:before {
-            border-left: 1px solid #666;
-        }
-
-        .jw-icon-inline:first-child:before {
-            border: none;
-        }
-    }
+    .jw-playlist-container {
+    left: -38%;
+  }
 
-    /* Icons, text and container styles for the dock, in the upper right corner */
-    .jw-dock {
-        .jw-dock-button {
-            border-radius: 50%;
-            border: 1px solid #333;
-        }
-        .jw-overlay {
-            border-radius: 2.5em;
-        }
-    }
-    
-    /* Styles for active list items */
-    .jw-icon-tooltip {
-        .jw-active-option {
-            background-color: @active-color;
-            color: #fff;
-        }
-    }
-    
-    /* A min-width on the volume icon to keep the width from collapsing on toggle */
-    .jw-icon-volume {
-        min-width: 2.6em;
-    }
-    
-    /* For the overlay containers */
-    .jw-time-tip,
-    .jw-menu,
-    .jw-volume-tip,
-    .jw-skip {
-        border: 1px solid #333;
-    }
-    
-    /* Position and padding of slider thumbnail */
-    .jw-time-tip {
-        padding: 0.2em;
-        bottom: 1.3em;
+  .jw-time-tip {
+    padding: 0.2em;
+    .jw-text {
+      color: @accent-color;
     }
+  }
 
-    /* Position of remaining overlay containers */
-    .jw-menu,
-    .jw-volume-tip {
-        bottom: .24em;
-    }
-    
-    /* Skip icon and text styles */
-    .jw-skip {
-        padding: 0.4em;
-        border-radius: 1.75em;
+  .jw-active-option {
+    background-color: rgba(255, 255, 255, 0.1);
+  }
 
-        .jw-text,
-        .jw-icon-inline {
-            color: @inactive-color;
-            line-height: 1.75em;
-        }
+  .jw-skip {
+    border-radius: 1em;
+  }
 
-        &.jw-skippable:hover .jw-text,
-        &.jw-skippable:hover .jw-icon-inline {
-            color: @active-color;
-        }
+  .jw-slider-vertical {
+    bottom: 0;
+    .jw-slider-container{
+      background-color: @rail-color;
     }
+  }
 
-    &.jw-flag-touch {
-        .jw-controlbar .jw-icon:before,
-        .jw-text-elapsed,
-        .jw-text-duration {
-            padding: 0 .35em;
-        }
 
-        .jw-controlbar .jw-icon-prev:before {
-            padding: 0 0.125em 0 0.7em;
-        }
+  .jw-display-icon-container {
+    display: table;
+    border: 4px solid @accent-color;
+    font-size: 20px;
+  }
 
-        .jw-controlbar .jw-icon-next:before {
-            padding: 0 0.7em 0 0.125em;
-        }
+  .jw-display-icon-container > .jw-icon {
+    padding-left: 4px;
```

Comment:
This makes the replay and buffering icons look off-center. If it's intended to better align the play button only apply this to that icon or player state.